support-frontend/test/controllers/PricesControllerSerialisationTest.scala

```diff
@@ -31,6 +31,8 @@ class PricesControllerSerialisationTest extends AnyFlatSpec with Matchers {
       None,
       Some(IntroductoryPriceBenefit(0.5, 3, Issue)),
       None,
+      new DateTime("2020-01-01"),
```

Comment:
Why this particular date? Should we place this in a global const?

Reply:
It's just a date, it doesn't affect anything about the test. What would you see as a benefit of putting it in a `val`?

Reply:
Would guess not a lot to gain other than lining it up with tests below as well.